packages/babel-core/test/async.js

```diff
@@ -7,6 +9,21 @@ const nodeGte8 = (...args) => {
   testFn(...args);
 };
 
+async function compileInSpawedProcess(async) {
+  // !!!! hack is coming !!!!
+  // todo(Babel 8): remove this section when https://github.com/facebook/jest/issues/9430 is resolved
+  // We don't check process.versions.node here, it will fail if node does not support esm
+  // please publish Babel on a modernized node :)
+  const { stdout, stderr } = await util.promisify(cp.execFile)(
+    require.resolve(`./fixtures/babel-compile-${async ? "async" : "sync"}.mjs`),
+    { env: process.env },
```

Comment:
nit: By default `env` is passed with `process.env`. https://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback

Reply:
Actually, I need someone with macOS to check if I need to do https://github.com/babel/babel/blame/31396b286daadee3c25c4891c8dd31a92d7f0604/packages/babel-core/test/config-chain.js#L63 for this test.

Reply:
`make prepublish` is passing as of dfa75fa65f on macOS.